src/docs/product/security/mobile-privacy.mdx

```diff
@@ -54,3 +54,12 @@ No. Sentry does not cause your app to contain any ads.
 ### What do I need to disclose to Google Play?
 
 You would need to disclose and/or include in your app privacy policy all types of data you are collecting through your app, including data you are sending to Sentry.
+
+## Sentry SDKs for third-party SDKs and libraries
+
+Sentry SDKs are not officially supported and recommended for third-party SDKs and libraries due to some technical reasons.
+
+- Sentry SDKs are not designed to be used in third-party SDKs and libraries.
+- Version conflicts may occur between Sentry SDKs.
```

Comment:
```suggestion
- Version conflicts may occur if multiple Sentry SDKs are installed in the same application.
```

Reply:
Not sure if it's needed, the title should be clear about that, otherwise, I'd apply this to the title.
Every item of the bullet point list is about having multiple Sentry SDKs in the same application.

Reply:
Actually, maybe it makes sense to apply this sentence to the 2nd and 3rd bullet points.
the 1st and 4th items are not specific to multiple sentry SDKs in the same application.